src/service/db/dashboard.rs

```diff
@@ -13,37 +13,46 @@
 // limitations under the License.
 
 use anyhow::Context as _;
+use tracing::instrument;
 
 use crate::{
     common::json,
     meta::dashboards::{Dashboard, NamedDashboard},
 };
 
-pub async fn get(org_id: &str, name: &str) -> Result<Option<NamedDashboard>, anyhow::Error> {
+/// # Panics
+///
+/// Panics if [`Dashboard::dashboard_id`] stored in the database is not equal to
+/// `dashboard_id` argument.
+#[instrument(err)]
+pub async fn get(
+    org_id: &str,
+    dashboard_id: &str,
+) -> Result<Option<NamedDashboard>, anyhow::Error> {
```

Comment:
I'm going to change the returned type to `Result<Dashboard, anyhow::Error>` in the upcoming commit.